OpenSim/Tools/MarkerPlacer.cpp

```diff
@@ -242,17 +242,17 @@ bool MarkerPlacer::processModel(Model* aModel, const string& aPathToSubject)
     /* Load the static pose marker file, and average all the
      * frames in the user-specified time range.
      */
-    MarkerData staticPose(aPathToSubject + _markerFileName);
+    MarkerData* staticPose = new MarkerData(aPathToSubject + _markerFileName);
```

Comment:
IMO It's slightly better to use a unique_ptr and then release it when you pass this pointer onto the `MarkersReference`. The reason is that the memory is still leaked if an exception is thrown before you construct the `MarkersReference` (for example, 2 lines down, there's an exception).

```cpp
std::unique_ptr<MarkerData> staticPose(new MarkerData(...));
...
MarkersReference markersReference(staticPose.release(), &markerWeightSet);
```